src/sass/components/_faq.scss

```
@@ -0,0 +1,131 @@
+
+
+.faq-content{     
+   margin: 0 auto;
+   width: 70%;
+}
+
+.u-list-reset {
+  list-style: none;
+  
+   margin:{
+      top: 0;
+      bottom: 0;    
+  } 
+  padding-left: 0;
+}
+
+#question-spacing{
+  padding-bottom: 25px;
+}
+
+
+.faq-set{
+  margin-top: 100px;  
+}
+
+
+
+.faq-header{
+  font-size: 35px;
+  text-align: center;
+  text-decoration: underline;
+  
+  
+  @media (max-width: 600px) {
+    font-size: 24px;
+    }   
+}
+
+
+
+input[type=text], select, textarea {
+  border: 1px solid #ccc;
+  border-radius: 4px;
+  box-sizing: border-box; 
+  
+  margin:{
+    top: 6px;
+    bottom: 16px;      
+  }
+  
+  padding: 12px;    
+  resize: vertical;  
+  width: 100%;
+  
+}
+
+input[type=submit] {
+  background-color: green;border: none;
+  border-radius: 4px;  
+  color: white;
+  cursor: pointer;
+  padding: 12px 20px;
+  width: 100%;  
+  
+  &:hover {
+     opacity: 70%;
+  }
+  
+}
+
+
+.form-header{
+  margin: 0 auto;
+  
+  padding: {
+    bottom: 15px;
+    left: 20px;
+    top: 50px;    
+  }
+  
+  width: 40%;
+  
+  
+  
+    
+  @media (min-width: 600px) and (max-width: 900px) {
+    width: 80%;
+    }  
+  
+  @media (max-width: 600px) {
+    width: 70%;
+    }  
```

If you tackle this mobile first, you don't need as complex of media queries:
```suggestion
  width: 70%;
  
  @media (min-width: 600px) {
    width: 80%;
    }  
  
  @media (min-width: 901px) {
    width: 40%;
    }  
```